tests/Integrations/PHPRedis/PHPRedisProjectTestsuite/3.x/RedisArrayTest.php

```diff
@@ -0,0 +1,554 @@
+<?php defined('PHPREDIS_TESTRUN') or die("Use TestRedis.php to run tests!\n");
```

Comment:
Is there a license file we can include for these vendored files?

Reply:
just as a followup, this has been removed.